index.html

```diff
@@ -101,21 +105,26 @@ <h2 id="connect">Let's Connect</h2>
       </ul>
     </section>
     <section id="works-container">
```

Comment:
- [ ] [ **OPTIONAL** ] APPLICABLE FOR DESKTOP VERSION: The horizontal spacing seems to be too much around the work image compared to Figma (highlighted in the screenshot). Please kindly stick to the Figma template 😃 Additionally, if you notice carefully in the Figma template, the skills tags (html, css , javascript) are rounded. You can add `border-radius` for the same in your design to make it similar to the Figma template 😄 

Your design 👇🏽 
<img width="1440" alt="Screenshot 2022-12-10 at 1 25 04 AM" src="https://user-images.githubusercontent.com/24832846/206786175-7ca700cb-c0c3-4baa-b8c4-63a440740498.png">


Figma design 👇🏽 
<img width="1129" alt="Screenshot 2022-12-10 at 1 30 21 AM" src="https://user-images.githubusercontent.com/24832846/206786709-069291f9-da43-4499-9cf1-a2068d7529e4.png">